### PR TITLE
Update to github.com/golang-jwt/jwt/v5

### PR DESCRIPTION
**Pull request by LimJiAn**
Summary

Update https://github.com/dgrijalva/jwt-go to https://github.com/golang-jwt/jwt.
Because of THIS REPOSITORY IS NO LONGER MAINTANED, it looks good to change it.

Ticket Link

None

Screenshots

None

Release Note

None

### CODE REVIEW

**Comment by agnivade** [comment]
@LimJiAn - Could you please sign the CLA? Then we can go ahead with the review. Thanks.

**Comment by LimJiAn** [comment]
> @LimJiAn - Could you please sign the CLA? Then we can go ahead with the review. Thanks.

@agnivade 
CLA signature completed. Thank you.

File: server/channels/app/platform/license.go [hunk 1]
```diff
@@ -353,7 +353,7 @@ func (ps *PlatformService) GenerateRenewalToken(expiration time.Duration) (strin
 	claims := &JWTClaims{
 		LicenseID:   license.Id,
 		ActiveUsers: activeUsers,
-		StandardClaims: jwt.StandardClaims{
+		RegisteredClaims: jwt.RegisteredClaims{
 			ExpiresAt: expirationTime.Unix(),
```
Comment (jupenur) [inline server/channels/app/platform/license.go:357]:
```suggestion
			ExpiresAt: jwt.NewNumericDate(expirationTime),
```

The type of the `ExpiresAt` field changed compared to the same field in the old `StandardClaims` struct.

**Review by jupenur — CHANGES_REQUESTED** [review]
This looks like a welcome change. Let's just get the CI to pass 🙂

File: server/channels/app/platform/license.go (hunk 1 above)
Comment (LimJiAn) [reply server/channels/app/platform/license.go]: Thank you for checking and changing the code.

**Comment by agnivade** [comment]
Hey @saturninoabril - The test link comes up as blank for me? Is there anything broken?

**Comment by saturninoabril** [comment]
I'll take a look. I can see the report posted in community channel - https://community.mattermost.com/core/pl/9madp96re7du5k6hakki6mmo9e. Likely there's something wrong in the dashboard.

**Review by agnivade — APPROVED** [review]

**Comment by agnivade** [comment]
Thanks @LimJiAn !

**Pull request merged**